docs/DeveloperGuide.md

```diff
@@ -2,37 +2,254 @@
+`showAccountMenu()` is used to display the list of choices to the user.
+`getUserSymptoms` is to ask input form user and store it in an arraylist.
+`addSymptomToSymptomList()` is used to add the symptom to the symptom list.
+`parseSymptomInput()` is used to parse the symptom input from the user.
+`addSymptoms()` is used to add the symptoms to the symptom list.
+`displaySymptomsList()` is used to display the list of symptoms to the user.
+`displayPossibleIllnesses()` is used to display the list of possible illnesses to the user.
+
+### Information
+The information class is responsible for displaying information to the user. It is responsible for displaying the
+information of the patient, the medicine and the diagnosis. It is also responsible for displaying the list of
+symptoms and the list of illnesses. A hashmap is used to store the information of the patient and the hashed password.
+
+`storePatientInfo()` is used to store the patient information into the hashmap. 
+`getPatientInfo()` is used to retrieve the patient information from the hashmap.
+`printDiagnosis()` is used to print the diagnosis of the patient.
+`resetDiagnosticHistory()` is used to reset and clear the diagnostic history of the patient.
+`checkHash()` is used to check if the hashed password is correct and matches.
+`getAllPatientData()` is used to retrieve all the patient data from the hashmap.
+`resetSymptomChoice()` is used to reset the symptom choice of the patient in case an error was made.
+`hashPassword()` is used to hash the password of the patient.
+
+### Parser
+The parser class is responsible for parsing the user input. It is responsible for parsing the user input from the
+menu and the symptom list. It is also responsible for parsing the user input from the medicine list.
+
+`parseWelcome()` is used to parse the user input from the welcome menu.
+`parseAccountCommand()` is used to parse the user command and then execute the command based on the choice number.
+
+[comment]: <> (//@@author Brennanzuz)
+### Diagnosis
+
+![DiagnosisClassDiagram.png](diagrams%2FDiagnosisClassDiagram.png)
+
+#### Symptoms
+
+All possible symptoms that patients may suffer from are included into this enumeration.
+
+#### Illness
+
+This class is inherited by all other illnesses. It contains the name of the illness `illnessName`, its `severityLevel`,
+whether it is chronic `isChronic`, and most importantly, the list of symptoms specific to the illness `symptoms`.
+
+Child classes of `Illness` are given their values on creation of the object in their constructor function.
+Hence `new Diarrhoea()` will create an object with `.getSymptoms()` returning an `ArrayList<Symptom>`
+of `Symptom.WET_STOOL`, `Symptom.STOMACH_ACHE`, and `Symptom.DIARRHOEA`.
+
+Knowing this, each child object of `Illness` will be created and stored in `ArrayList<Illness> ALL_ILLNESSES`.
+
+#### Diagnosis algorithm
+
+![DiagnosisSequenceDiagram.png](diagrams%2FDiagnosisSequenceDiagram.png)
```

Comment:
For the medicine Manager, perhaps the activation bar should be closed or labelled as closed